packages/api/cli/src/electron-forge-start.ts

```diff
@@ -2,15 +2,31 @@ import path from 'path';
 
 import { api, StartOptions } from '@electron-forge/core';
 import { ElectronProcess } from '@electron-forge/shared-types';
+import chalk from 'chalk';
 import program from 'commander';
 import fs from 'fs-extra';
+import semver from 'semver';
+// eslint-disable-next-line node/no-unpublished-import
+import updateNotifier from 'update-notifier';
 
 import './util/terminate';
 import workingDir from './util/working-dir';
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const metadata = require('../package.json');
```

Comment:
issue: This can be converted to an `import` statement to avoid the ESLint error.